custom_components/powercalc/data/signify/LCX004/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.1:docker"
+    },
+    "name": "LCX004-extended",
```

```suggestion
    "name": "Hue Gradient Lightstrip   ",
```

This information will be used when generating the supported models listing.